typescript/sdk/src/app/MultiProtocolApp.ts

```diff
@@ -54,33 +50,32 @@ export class BaseSealevelAdapter<
  * @param multiProvider - A MultiProtocolProvider instance that MUST include the app's
  *   contract addresses in its chain metadata
  * @param logger - A logger instance
+ *
+ * @override protocolToAdapter - This should return an Adapter class for a given protocol type
  */
 export abstract class MultiProtocolApp<
   ContractAddrs = {},
   IAdapterApi extends BaseAppAdapter = BaseAppAdapter,
 > extends MultiGeneric<ChainMetadata<ContractAddrs>> {
-  // Subclasses should override this with more specific adapters
-  public readonly protocolToAdapter: AdapterProtocolMap<any, BaseAppAdapter> = {
-    [ProtocolType.Ethereum]: BaseEvmAdapter,
-    [ProtocolType.Sealevel]: BaseSealevelAdapter,
-  };
-
   constructor(
     public readonly multiProvider: MultiProtocolProvider<ContractAddrs>,
     public readonly logger = debug('hyperlane:MultiProtocolApp'),
   ) {
     super(multiProvider.metadata);
   }
 
+  // Subclasses should override this with more specific adapters
+  abstract protocolToAdapter(
```

Comment:
Changing this from a member var to a method because of tricky errors in subclasses due to generic types